### ISSUE TITLE
Error using sample code

### ISSUE THREAD

**Issue by johnfmorton** [KuriousAgency/craft-qrcode#4]
The code sample that passes in an array throw errors for me. 

```
{{ {"name":"John Smith"}|qrcode(100) }}
```

or 

```
{{ craft.qrcode.generate({"name":"John Smith"}, 100) }}
```

The error I receive is:

```
Argument 1 passed to Endroid\QrCode\QrCode::__construct() must be of the type string, array given, called in /Users/john/Sites/mytestsite/vendor/kuriousagency/qrcode/src/services/QRCodeService.php on line 36
```

Passing a simple string does work though. The following generates no errors.

```
{{ craft.qrcode.generate(1234567890) }}
```

I'm unclear on the benefit of passing an array but wondered if there is something broken since the example code isn't generating a QR code.

**Comment by samuelbirch** [comment]
thanks, i'll take a look.

**Comment by samuelbirch** [comment]
This is now fixed in 1.0.3

The first release json encoded all data, but that wasn't suitable for numbers. So encoding was removed but the examples weren't updated. So i've changed it to that is an array is passed in then it will json encode it.
